etc/conda-install.sh

```diff
@@ -89,9 +88,11 @@ elif [ -n "${CHANGED// }" ]; then
     # print the diff to the console for debugging
     diff -wy $OUTSIDE_ENV_YML $INSIDE__ENV_YML || true
     # update the existing environment
-    mamba update -n base -c defaults conda
-    mamba update -n base -c conda-forge mamba
-    mamba env update -n $ENV_NAME --file $INSIDE__ENV_YML --prune || CONDA_ENV_UPDATE_FAILED=1
+    mamba update -n base -c conda-forge mamba \
+    && mamba update -n base -c conda-forge conda \
```

Comment:
Same suggestion here:

```suggestion
    mamba update -n base mamba conda \
```